main.rb

```diff
@@ -1,6 +1,11 @@
 class Main
+  def initialize
+    @word = 'Hello world!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.